activerecord/test/cases/validations/uniqueness_validation_test.rb

```diff
@@ -5,6 +5,7 @@
 require 'models/warehouse_thing'
 require 'models/guid'
 require 'models/event'
+require 'models/matey'
```

Comment:
How about we just set the table name on the subclass, rather than inheriting from this model, so we don't need to require more model files?